### ISSUE TITLE
Datalink: Access to lower data plane in datalink variables

### ISSUE THREAD

**Issue by gelicia** [grafana/grafana#82746]
### Discussed in https://github.com/grafana/grafana/discussions/67355

<div type='discussions-op-text'>

<sup>Originally posted by **philippexe** April 27, 2023</sup>
This is mainly observed on a Pie Chart, but probably generalizes to other visualizations aswell.

It would be very helpful to be able the access the data plane behind the visualization calculation layer. 
Example, we have a version pie chart displaying the aggregated versions of the systems in the field. Now if an old version is observed one would like to take action on it. The data of the system ids is available to the pie chart, in terms of the query A returning it. The Piechart then uses the Value Option -> Calculation for the aggregation. Now we have a datalink to a more detailed systems overview, but for that access the actual system ids is needed in the datalink. Currently only ${__value.raw} and similiar variables are available, but those all refer to the value state after the calculation.

<img width="379" alt="Bildschirm­foto 2023-04-27 um 10 19 03" src="https://user-images.githubusercontent.com/24919839/234802753-935f07fb-abdd-4b80-9a6e-ea50458ab29f.png">

In this case the label is e.g. 3.7.2. where ${__value.raw}  now refers to the value 2 and ${__field.labels._value} gives 3.7.2.
But in Inspect -> Data we can see that the Panel is totally aware of the underlying system ids. A variable referencing those would be needed e.g. ${__data.value}

<img width="597" alt="Bildschirm­foto 2023-04-27 um 10 14 24" src="https://user-images.githubusercontent.com/24919839/234801601-877ed57a-10c9-4799-8e99-59d682cc747a.png">

TLDR;
New Datalink variable to access lower dataplane (Inspect -> Data)</div>
